doc/control-styles.md

```diff
@@ -0,0 +1,48 @@
+# Control Styles
+Control|Style Key|Implicit Style*|
+-|-|-
+AppBarButton|AppBarButtonStyle|implicit
+Button|ElevatedButtonStyle|
+Button|FilledButtonStyle|implicit
+Button|FilledTonalButtonStyle|
+Button|OutlinedButtonStyle|
+Button|TextButtonStyle|
+Button|IconButtonStyle|
+CalendarDatePicker|CalendarDatePickerStyle|implicit
+CalendarView|CalendarViewStyle|implicit
+CheckBox|CheckBoxStyle|implicit
+CommandBar|CommandBarStyle|implicit
+DatePicker|DatePickerStyle|implicit
+FlyoutPresenter|FlyoutPresenterStyle|implicit
+ListView|ListViewStyle|implicit
+ListViewItem|ListViewItemStyle|implicit
+MenuFlyoutPresenter|MenuFlyoutPresenterStyle|implicit
+muxc:NavigationView|NavigationViewStyle|implicit
+muxc:NavigationViewItem|NavigationViewItemStyle|implicit
+PasswordBox|FilledPasswordBoxStyle|implicit
+PasswordBox|OutlinedPasswordBoxStyle|
+RadioButton|RadioButtonStyle|implicit
+TextBlock|DisplayLarge|
+TextBlock|DisplayMedium|
+TextBlock|DisplaySmall|
+TextBlock|HeadlineLarge|
+TextBlock|HeadlineMedium|
+TextBlock|HeadlineSmall|
+TextBlock|TitleLarge|
+TextBlock|TitleMedium|
+TextBlock|TitleSmall|
+TextBlock|LabelLarge|
+TextBlock|LabelMedium|
+TextBlock|LabelSmall|
+TextBlock|BodyLarge|
+TextBlock|BodyMedium|implicit
+TextBlock|BodySmall|
+TextBox|FilledTextBoxStyle|implicit
+TextBox|OutlinedTextBoxStyle|
+ToggleButton|TextToggleButtonStyle|implicit
+ToggleButton|IconToggleButtonStyle|
+
+Implicit Style*: Styles marked with `implicit` can be made into the default implicit styles by setting the `WithImplicitStyles` flag to `implicit`:
```

Comment:
It should be setting the flag to `true`, not `implicit`. Unless something has changed?

Reply:
i blame search-replace-all!